frame/contracts/src/weights.rs

```diff
@@ -18,22 +18,22 @@
 //! Autogenerated weights for pallet_contracts
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-06-22, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
-//! HOSTNAME: `bm3`, CPU: `Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz`
+//! DATE: 2022-07-07, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `test-bench-bot`, CPU: `Intel(R) Xeon(R) CPU @ 3.10GHz`
```

Comment:
The bench bot is using the wrong hardware @joao-paulo-parity.  
It should be the same as before.

Reply:
I don't know why that happened since the job is using `bench-bot`, not `test-bench-bot`, as its tag. I'll open a ticket for CI/CD.

Reply:
Created https://github.com/paritytech/ci_cd/issues/518. Thanks for noticing this.